src/interval.jl

```diff
@@ -71,6 +71,20 @@ struct Interval{T} <: AbstractInterval{T}
     end
 end
 
+"""
+Check if a value is ∞ or not. Specifically check if the type is `Infinite`,
+or `InfExtended`, and then check if the value is pos or neg infinity.
+"""
+function isbounded(a)
+    T = typeof(a)
+    if T <: Infinite || T <: InfExtended
+        return !isposinf(a) && !isneginf(a)
+    else
+        return true
+    end
+end
```

Comment:
`isfinite` works with `Infinity` and `InfExtended` types as well as the `Inf` from `Float64`. Additionally this works with `TimeType`s:
```suggestion
isbounded(x) = isfinite(x)
```